packages/web/docs/src/pages/features/publish-schema.mdx

```diff
@@ -21,6 +21,8 @@ Install [`@graphql-hive/cli`](https://www.npmjs.com/package/@graphql-hive/cli) p
 
 Install latest version:
 
+<Callout type="warning">You need TLS v1.2 or higher to install this script.</Callout>
```

Comment:
I feel like it might be clear on "how do I check I have TLS 1.2" or "what do I do if I don't have it"?

Reply:
It is kinda tricky because if you just do `curl --version` on Mac it may not tell you what TLS protocols are supported directly but it uses SecureTransport which supports it. 
![CleanShot 2023-03-03 at 09 44 33](https://user-images.githubusercontent.com/44710980/222749972-27fad936-c788-42c6-81fa-713a30aa516d.png)

Reply:
I think it would be convenient to include this in the docs.

I believe that was what @dotansimha  was suggesting... Instead of just displaying a callout warning, 
have this information included in a Prerequisite/ Requirement section of the docs and have the callout warning link to that section.

Love your work @saihaj  💖